### DISCUSSION TITLE
GitHub job runs indefinitely in the CI when using task distribution with the cloud

### DISCUSSION THREAD

**Discussion by edbzn** [nrwl/nx#7889] [General]
Hi, 

We have struggles in our repository with a [target running indefinitely in the CI using Nx Cloud](https://github.com/rx-angular/rx-angular/runs/4324883320?check_suite_focus=true), I suspect Nx Cloud agents not handling multiple jobs running in bands, but I'm not totally sure. Here is [the definition of the job](https://github.com/rx-angular/rx-angular/blob/cab59b1e86228fa258280bc7954eeabdef019388/.github/workflows/build-and-test.yml#L56). The CI hangs on the following task `build-docs` job, this job requires the `build` job to successfully run before, maybe this is a scenario not handled by the Nx Cloud.

Locally it's working well, so I'm going to disable the cloud task distribution for this particular job. Any inputs on this issue?

Thanks in advance.

**Comment by cofenster-tiago** [comment]
Did you ever find out the reason?

**Reply by TalMikey** [reply]
do you have a solution?
